guides/Front-end-strategy.md

```diff
@@ -0,0 +1,44 @@
+# Front-end strategy
+
+### What we use
+####Foundation 6 framework
+Use only required components and libraries. We don't need to use normalize with Foundation 6, it included in foundation-global-styles mixin.
+
+####Grunt
+Use grant tasks for creating svg sprites
+
+####Autoprefixer
+Use Autoprefixer gem or Autoprefixer of [grunt-postcss](https://github.com/nDmitry/grunt-postcss) task in Shopify projects.
+
+####Bower
+Use bower with bower installer for manage libraries and control their versions.
+
+####Front-man
+It includes utilities classes, z-index variables and helpers for sprite and image tags.
```

Comment:
"Utility classes"

How do we use it? Is this a command to run on a terminal? A Library to include? Where and how do we include it if so?